lib/ClangImporter/ClangImporter.cpp

```diff
@@ -5207,37 +5207,48 @@ TinyPtrVector<ValueDecl *> ClangRecordMemberLookup::evaluate(
         continue;
 
       auto *baseRecord = baseType->getAs<clang::RecordType>()->getDecl();
-      if (auto import = clangModuleLoader->importDeclDirectly(baseRecord)) {
+      if (auto baseRecordName = clangModuleLoader->importName(baseRecord)) {
```

Comment:
This is the main change in this PR: we no longer import all of the base structs during a name lookup request.